controllers/addinvoice.ctrl.go

```diff
@@ -61,7 +61,7 @@ func AddInvoice(c echo.Context, svc *service.LndhubService, userID int64) error
 	}
 
 	amount, err := svc.ParseInt(body.Amount)
-	if err != nil {
```

Comment:
we need to check this here because we accept strings as request body?

Reply:
The validate tag won't work because it's a string right? So we have to check seperately

Reply:
true, thanks.